sdk/tables/Azure.Data.Tables/src/TableInternalRestClient.cs

```diff
@@ -349,5 +349,41 @@ internal HttpMessage CreateDeleteEntityRequest(string table, string partitionKey
             }
             return message;
         }
+
+        internal HttpMessage CreateSetAccessPolicyRequest(string table, int? timeout, string requestId, IEnumerable<SignedIdentifier> tableAcl)
```

Comment:
Can you please add a comment why this is required.

Reply:
I didn't intend to leave this in. This was from experimentation to figure out why these requests were being rejected by the service for malformed xml - thanks for catching it.